.github/workflows/pull-request.yml

```
@@ -348,3 +348,50 @@
+        run: docker exec talawa-api-1 /bin/bash -c 'nohup pnpm run start_development_server > /dev/null 2>&1 &'
+      - name: Wait for GraphQL endpoint to become available
+        run: |
+          echo "Waiting for the GraphQL endpoint to become available..."
+          for i in {1..30}; do
+            # Check if container exists first
+            if ! docker ps | grep -q talawa-api-1; then
+              echo "Container talawa-api-1 not found. Waiting..."
+              sleep 2
+              continue
+            fi
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)